woocommerce/woocommerce-gutenberg-products-block
# Move icon components from SearchList to a app-wide file

This PR moves the checkmark icons to a new component, which will serve as a collection for all future custom SVG icons. This also adds the `new_releases` icon from Material, for use in the New Products block #255. There should be no visual changes.

**To test**

- Add a Products By Category block, look at the category list
- Expect: the checkmarks in the category list should appear as normal, no change.
- Check both the edit mode & the inspector, and both checked and unchecked.
